lib/shared/addon/components/form-ingress-backends/component.js

```diff
@@ -43,7 +43,7 @@ export default Component.extend({
         pathArray.pushObject(get(this, 'store').createRecord({
           type: 'httpingresspath',
           backendType: 'service',
-          targetPort: get(path, 'targetPort'),
+          targetPort: get(path, 'targetPort').toString(),
           serviceId: get(path, 'serviceId').replace('/', ':'),
```

Comment:
This is not new and I don't know what it's doing, but it's probably very wrong.  If the UI is still generating `$stack/$service` name that should be corrected to return the right ID rather than trying to string replace into a valid one.